### PR TITLE
Update contributing file

### PR DESCRIPTION
**Pull request by luisg5**
### Summary

Modified CONTRIBUTING.md file.

### Details and comments

Made modifications that fixed markup issues and were consistent with the CONTRIBUTING.md file within the qiskit-terra repository.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -130,41 +122,38 @@ Issue #190: Short summary of the issue
...
-
-We recommend using [Python virtual
-environments](https://docs.python.org/3/tutorial/venv.html) to cleanly
-separate Qiskit from other applications and improve your experience.
-:::
-
-Setup with an environment
--------------------------
+**Note:**
+> We recommend using [Python virtual environments](https://docs.python.org/3/tutorial/venv.html)
```
Comment (jyu00) [inline CONTRIBUTING.md]: To be consistent with Terra (which recommends Anaconda), it might be better to not recommend a specific environment. We can say something like
```We recommend using a self-contained environment, such as Python virtual environments or Anaconda environments, to cleanly separate Qiskit from other applications and improve your experience.```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -130,41 +122,38 @@ Issue #190: Short summary of the issue
...
-conda create -y -n QiskitDevenv python=3
-source activate QiskitDevenv
+$ conda create -y -n QiskitDevenv python=3
+$ conda activate QiskitDevenv
+```
+
+### Setup with Python virtual environment
+```{.sh}
+$ python -m venv QiskitDevenv
+$ source QiskitDevenv/bin/activate (Linux and Mac)
```
Comment (jyu00) [inline CONTRIBUTING.md]: Can you separate out Linux/Mac and Windows commands instead of adding the `()` in the end? It's more lines but clearer and more consistent.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -130,41 +122,51 @@ Issue #190: Short summary of the issue
...
 This section include some tips that will help you to push source code.
 
-::: {.note}
-::: {.admonition-title}
-Note
-:::
+**Note:**
+> We recommend using a self-contained environment, 
+> such as [Python virtual environments](https://docs.python.org/3/tutorial/venv.html) 
+> or Anaconda environments, to cleanly separate Qiskit from other applications
```
Comment (jyu00) [inline CONTRIBUTING.md]: Can you add a link to Anaconda? Preferably something that talks about setting up a virtual env (similar to the python vir env link)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -130,41 +122,51 @@ Issue #190: Short summary of the issue
...
+$ conda activate QiskitDevenv
+```
 
-The simplest way to use environments is by using Anaconda
+### Setup with Python virtual environment
 
-``` {.sh}
-conda create -y -n QiskitDevenv python=3
-source activate QiskitDevenv
+```{.sh}
```
Comment (jyu00) [inline CONTRIBUTING.md]: nit - can you add a sentence before each command to make it read more smoothly? maybe something like 

 Issue the following command to create a Python virtual environment:
```
<command>
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -129,30 +129,39 @@ This section include some tips that will help you to push source code.
...
 > and improve your experience.
 
 ### Setup with Conda virtual environment
 
+Issue the following command to create and activate the virtual environment:
+
 ``` {.sh}
 $ conda create -y -n QiskitDevenv python=3
-$ conda activate QiskitDevenv
+$ source activate QiskitDevenv
```
Comment (jyu00) [inline CONTRIBUTING.md]: why did you change `conda activate` to `source activate`? the latter is the newer way of activating the environment.

Comment (luisg5) [reply CONTRIBUTING.md]: On the documentation online, it suggested 'source activate ...'. But, it was for versions before 4.6. I will correct that.
